runtime/onert/backend/cpu/ops/GatherLayer.cc

```diff
@@ -82,15 +83,15 @@ void GatherLayer::runByGGMLQuantInputType()
     throw std::runtime_error("Gather: invalid indices tensor shape");
 
   if (_indices->data_type() != ir::DataType::INT32)
-    throw std::runtime_error("Gather: indices tensor must be int32 type");
+    throw UnsupportedDataTypeException{"Gather", _indices->data_type()};
 
   if (_axis != 0)
     throw std::runtime_error("Gather: axis must be 0");
 
   // convert tensor
-  auto input = getGGMLTensor(_input);
-  auto indices = getGGMLTensor(_indices);
-  auto output = getGGMLTensor(_output);
+  auto input = getGGMLTensor("FullyConnected", _input);
```

Comment:
Perhaps I'm missing something but why is this used with `FullyConnected` op name?